index.html

```diff
@@ -8,14 +8,19 @@
     <link rel="stylesheet" href="index.css" />
 </head>
 <body onload="bookManager.loadData()">

```

Comment:
- [ ] [OPTIONAL] I see that you have called event handlers as inline attributes in HTML, I would say to do all javascript event handlers and function calls in the js file it is generally considered bad practice to use inline attributes for event handlers [you can read more about it here ](https://stackoverflow.com/questions/11737873/why-are-inline-event-handler-attributes-a-bad-idea-in-modern-semantic-html) I would recommend you to fix this

Reply:
Done.